index.html

```diff
@@ -295,6 +266,18 @@ <h4>View only</h4>
 					state="download"
 					size="medium"
 				></ts-file-card>
+
+				<script>
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') The length of this JS script (10) exceeds the maximum set to 5. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=Web%3ALongJavaScriptCheck)